doc/stdenv/stdenv.chapter.md

```diff
@@ -116,6 +116,65 @@ On Linux, `stdenv` also includes the `patchelf` utility.
 
 ## Specifying dependencies {#ssec-stdenv-dependencies}
 
+Build systems often require more dependencies than just what `stdenv` provides. This section describes attributes accepted by `stdenv.mkDerivation` that can be used to make these dependencies available to the build system.
+
+### Simplified overview {#ssec-stdenv-dependencies-overview}
+A full reference of the many kinds of dependencies needed to cover all possible cases is provided in the next section, but here is a simplified overview of the most common kinds. It should cover most derivations.
+
+Add dependencies to `nativeBuildInputs` if they are executed during the build:
```

Comment:
```suggestion
Add dependencies to `nativeBuildInputs` if they are needed during the build:
```

These may be just files, not necessarily executables.

Reply:
The issue is that runtime libraries are also needed during the build (for linking).

Reply:
So saying "needed" is less wrong, isn't it?

Reply:
We do not want to add the runtime libraries to `nativeBuildInputs`, even though they are already “needed” for linking.

Reply:
Okay, what's the right phrasing then? Dependencies executed during the build sounds weird, because dependencies are generic and may or may not even contain executables.

Reply:
Maybe something like “if their code is executed” then? If the dependency does not contain code, it should not really matter where it is placed anyway.

Reply:
I think "executed" is correct, because all these distinctions are for cross: only executable files differ by architecture and matter for cross. An image can be in whatever it does not really matter.